core/test/Test.re

```diff
@@ -773,16 +757,278 @@ let suites = [
              [<Text id=2 title="y" />, <Text id=1 title="x" />],
              Some({
                subtreeChange: `PrependElement([<Text id=2 title="y" />]),
-               updateLog: ref([]),
+               updateLog: [],
              }),
            ),
          )
-      |> done_;
+      |> ignore;
+    },
+  ),
+];
+
+let mountLog = [
+  (
+    "Test rendered element mount",
+    `Quick,
+    () => {
+      let root = Implementation.{name: "root", element: View};
+
+      TestRenderer.render(<Components.BoxWrapper />)
+      |> mount(root)
+      |> expectHost(
```

Comment:
How about we put those asserts in a separate module and have simple `expect` there?